### ISSUE TITLE
Error while generating BRAT datasets

### ISSUE THREAD

**Issue by idalr** [ArneBinder/pie-datasets#83]
I encountered an issue while trying to run tests in `test_sciarg.py` when getting the `dataset` using `@pytest.fixture`, e.g. when running `test_sciarg.py::test_hf_dataset`, etc.

The error appeared as: 
```
E       UnicodeDecodeError: 'charmap' codec can't decode byte 0x9c in position 2017: character maps to <undefined>
...
E       datasets.builder.DatasetGenerationError: An error occurred while generating the dataset
```
and the full error message was [brat_error.txt](https://github.com/ArneBinder/pie-datasets/files/13600102/brat_error.txt).

I have further checked with the other BRAT datasets, i.e., `abstrct`, `aae2`, and also got the same error while running tests as well.

**Comment by idalr** [comment]
Fixed in [844de61](https://huggingface.co/datasets/DFKI-SLT/brat/commit/844de61e8a00dc6a93fc29dc185f6e617131fbf1)

**Comment by ArneBinder** [comment]
Thanks!
